prac_05/emails.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404 Practical
+Store names from email input in dictionary
+"""
+
+emails_dict = {}
+
+email = input("Email: ")
+while email != "":
+    email_alias = email.split('@')
+    name_list = email_alias[0].split('.')
+    name = ' '.join([word for word in name_list]).title()
+    choice = input("Is your name {}? (Y/n) ".format(name))
+    if choice.lower() == 'y':
+        emails_dict[email] = name
+    else:
```

Comment:
If any letter is selected apart from "y," is it intuitive that the result is "n?"